test/functional/test_framework/test_node.py

```diff
@@ -63,10 +73,12 @@ def __init__(self, i, dirname, extra_args, rpchost, timewait, binary, stderr, mo
         self.url = None
         self.log = logging.getLogger('TestFramework.node%d' % i)
 
-    def __getattr__(self, *args, **kwargs):
+        self.p2ps = []
+
+    def __getattr__(self, name):
         """Dispatches any unrecognised messages to the RPC connection."""
         assert self.rpc_connected and self.rpc is not None, "Error: no RPC connection"
-        return self.rpc.__getattr__(*args, **kwargs)
+        return self.rpc.__getattr__(name)
 
```

Comment:
This change seems to be unrelated, so maybe revert or move to another commit.

Also, it would be better `return getattr(self.rpc, name)` here to avoid making assumptions about python rpc class (like that it even has a \_\_getattr__ method), and so any exceptions thrown will be properly reported.